data/json/npcs/Kindred/NPC_Brigitte_LaCroix.json

```diff
@@ -0,0 +1,505 @@
+      { "item": "bone_glue", "prob": 20 },
+      { "item": "bone_flute", "prob": 10 },
+      { "item": "bone_human", "prob": 10 },
+      { "item": "broth_bone", "prob": 60 },
+      { "item": "water_clean", "prob": 90 }
+    ]
+  },
+  {
+    "type": "mutation",
+    "id": "seer_mark",
+    "name": { "str": "Mark of the Seer" },
+    "points": 1,
+    "description": "A strange occult symbol carved into your hand by Brigitte LaCroix.",
+    "starting_trait": false,
+    "valid": false,
+    "purifiable": false
+  },
+  {
+    "type": "npc",
+    "id": "SEER_Brigitte_LaCroix",
+    "//": "A mysterious member of a doomsday cult.",
+    "name_unique": "Brigitte LaCroix",
+    "gender": "female",
+    "name_suffix": "Bone Seer",
+    "class": "NC_BONE_SEER",
+    "attitude": 7,
+    "mission": 3,
+    "chat": "TALK_BONE_SEER",
+    "faction": "no_faction",
+    "mission_offered": "MISSION_SEER_GATHER_BONE"
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_BONE_SEER",
+    "dynamic_line": {
+      "u_has_var": "talked_to_seer",
+      "type": "dialogue",
+      "context": "first_meeting",
+      "value": "yes",
+      "no": "You there.  Quiet down.  Can you hear it?  The song?",
+      "yes": {
+        "u_has_var": "asked_about_song",
+        "type": "dialogue",
+        "context": "song",
+        "value": "yes",
+        "no": "You're back.  Have you come to listen to the song?",
+        "yes": {
+          "u_has_trait": "seer_mark",
+          "no": "Traveller.",
+          "yes": { "u_female": true, "yes": "Greetings, sister.", "no": "Greetings, brother." }
+        }
+      }
+    },
+    "speaker_effect": { "effect": { "u_add_var": "talked_to_seer", "type": "dialogue", "context": "first_meeting", "value": "yes" } },
+    "responses": [
+      {
+        "text": "What song?",
+        "topic": "TALK_BONE_SEER_SONG",
+        "condition": { "not": { "u_has_var": "asked_about_song", "type": "dialogue", "context": "song", "value": "yes" } }
+      },
+      {
+        "text": "So about the songs…",
+        "topic": "TALK_BONE_SEER_SONG2",
+        "condition": { "u_has_var": "asked_about_song", "type": "dialogue", "context": "song", "value": "yes" }
+      },
+      {
+        "text": "Do others follow this belief as well?",
+        "topic": "TALK_BONE_SEER_OTHERS",
+        "condition": { "u_has_var": "asked_about_song", "type": "dialogue", "context": "song", "value": "yes" }
+      },
+      {
+        "text": "Can you tell me more about yourself?",
+        "topic": "TALK_BONE_SEER_ABOUT",
+        "condition": { "u_has_var": "asked_about_song", "type": "dialogue", "context": "song", "value": "yes" }
+      },
+      {
+        "text": "You mentioned some cycle before.  What does that mean?",
+        "topic": "TALK_BONE_SEER_CYCLE",
+        "condition": { "u_has_var": "asked_about_song", "type": "dialogue", "context": "song", "value": "yes" }
+      },
+      {
+        "text": "I've met Cooper.",
```

Comment:
This would come up as an option if you've met cooper but this is your first time meeting Brigitte. That seems weird.